femm.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+export WINEPREFIX="${HOME}/.local/share/femm42-flatpak"
+export WINEDLLOVERRIDES="mscoree,mshtml="
+
+POE_INSTALLER_NAME="femm42bin_x64_21Apr2019.exe"
+POE_SETUP="${WINEPREFIX}/${POE_INSTALLER_NAME}"
+POE_DOWNLOAD_URL='https://www.femm.info/wiki/Files/files.xml?action=download&file=femm42bin_x64_21Apr2019.exe'
+POE_RUN_CMD="${WINEPREFIX}/drive_c/femm42/bin/femm.exe"
+
+WINE="/app/bin/wine"
+
+XORG_LOG="/var/log/Xorg.0.log"
+
+VERSION_NUM="1.0.0"
+VERSION_FILE="${WINEPREFIX}/info.femm.unofficial"
+
+declare -ra WINE_PACKAGES=(directx9 corefonts tahoma win7)
+declare -ra WINE_SETTINGS=('csmt=on' 'glsl=disabled')
+
+echo "#############################################"
+echo "## FEMM 4.2 Unofficial Flatpak v${VERSION_NUM} ##"
+echo "#############################################"
+echo
+
+set_wine_settings(){
+  local my_documents="${WINEPREFIX}/drive_c/users/${USER}/My Documents"
+
+  echo "Installing wine requirements."
+  winetricks --unattended "${WINE_PACKAGES[@]}"
+
+  echo "Setting wine settings."
+  winetricks --unattended "${WINE_SETTINGS[@]}"
+
+  echo
+}
+
+# Run only if POE isn't installed
+first_run(){
+  set_wine_settings
+
+  echo "${VERSION_NUM}" > "${VERSION_FILE}"
+
+  if [ ! -f "${POE_SETUP}" ]; then
+    echo "Downloading FEMM 4.2."
+    curl -o "${POE_SETUP}" -O -J -L "${POE_DOWNLOAD_URL}" 
```

Comment:
you should make this file downloadad as "extra data" and run the install steps with `apply_extra`.

Reply:
Correct if I'm wrong, but if I do so, files will be kept inside sandbox. I need to expose the mfiles folder. Thus the software can interact with octave and python.